conda-store-server/conda_store_server/action/generate_lockfile.py

```diff
@@ -37,6 +37,16 @@ def print_cmd(cmd):
     print_cmd(["conda", "config", "--show"])
     print_cmd(["conda", "config", "--show-sources"])
 
+    # conda-lock ignores variables defined in the specification, so this code
+    # gets the value of CONDA_OVERRIDE_CUDA and passes it to conda-lock via
+    # the with_cuda parameter, see:
```

Comment:
Nit, so non blocking but since we said this is an "interim" solution it would be best to add also a TODO/something indicating that we should look at a more robust approach